### PR TITLE
added category id

### PR DESCRIPTION
**Pull request by AhsanBracu**

### CODE REVIEW

File: app/src/main/java/com/framgia/elsytem/CategoriesActivity.java [hunk 1]
```diff
@@ -55,6 +58,15 @@ protected void onCreate(Bundle savedInstanceState) {
         list = (ListView) findViewById(R.id.listCategory);
         okHttpClient = new OkHttpClient();
         new HttpAsyncCategory().execute("https://manh-nt.herokuapp.com/categories.json?auth_token=" + token);
+        list.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                int category_id = categoriesName.get(position).getId();
+                Intent intent = new Intent(getApplication(), QuestionActivity.class);
+                intent.putExtra("Category_id",category_id);
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/CategoriesActivity.java]: hardcode

File: app/src/main/java/com/framgia/elsytem/CategoriesActivity.java [hunk 2]
```diff
@@ -55,6 +58,15 @@ protected void onCreate(Bundle savedInstanceState) {
         list = (ListView) findViewById(R.id.listCategory);
         okHttpClient = new OkHttpClient();
         new HttpAsyncCategory().execute("https://manh-nt.herokuapp.com/categories.json?auth_token=" + token);
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/CategoriesActivity.java]: read my comment or ask mr Shawon Sharma for updating

File: app/src/main/java/com/framgia/elsytem/QuestionActivity.java [hunk 3]
```diff
@@ -103,7 +104,10 @@ public void onClick(View v) {
         Constants.PREF_STATE = getApplicationContext().getSharedPreferences("MyPref",
                 MODE_PRIVATE);
         word_number = Constants.PREF_STATE.getInt(String.valueOf(Constants.STATE), 0);
-        new HttpAsyncLesson().execute(" https://manh-nt.herokuapp.com/categories/1/lessons.json");
+        Intent intent=getIntent();
+        category_id=intent.getIntExtra("Category_id",1);
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/QuestionActivity.java]: hardcode

File: app/src/main/java/com/framgia/elsytem/QuestionActivity.java [hunk 4]
```diff
@@ -103,7 +104,10 @@ public void onClick(View v) {
...
+        Toast.makeText(this,"category id "+category_id,Toast.LENGTH_LONG).show();
+        new HttpAsyncLesson().execute(" https://manh-nt.herokuapp.com/categories/"+category_id+"/lessons.json");
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/QuestionActivity.java]: same as above

File: app/src/main/java/com/framgia/elsytem/CategoriesActivity.java (hunk 2 above)
Comment (AhsanBracu) [inline app/src/main/java/com/framgia/elsytem/CategoriesActivity.java]: OK

**Comment by nvmanh** [comment]
OK!

**Pull request merged**